ops/cloud-deployment/monitoring-alerts/alert-action-group.bicep

```diff
@@ -0,0 +1,48 @@
+param actionGroupName string = 'EmailDevelopmentTeam'
+
+resource actionGroup 'microsoft.insights/actionGroups@2023-01-01' = {
+  name: actionGroupName
+  location: 'Global'
+  properties: {
+    groupShortName: 'Email Team'
+    enabled: true
+    emailReceivers: [
+      {
```

Comment:
The reason to have an explicit list of emails: 
    if we were to use armRoles, it would send to the primary email on the user account
    our flexion emails are not the primary emails on these accounts

emails can be added as necessary here.

Reply:
Can we possibly create distribution group email and use that instead?

Reply:
So the armRoles primary email is the doj emails? Could we do both the armRoles and an explicit list of emails? I am thinking for Stan and Jaya. Also, I'd like to get alerts.

Reply:
I think creating one group and having multiple emails in it. At this point I can't imagine why we would leave anyone on the team off of it.

Reply:
I am in the process of testing out the action group sending alerts to specific subscription level arm roles. I will get everyone added appropriately if it functions as expected.